docs/se/guides/overview.adoc

```diff
@@ -67,6 +67,41 @@ Learn how to use Helidon SE built-in and application metrics.
 --
 Learn how to trace a Helidon SE application.
 --
+
+[CARD]
+.OIDC Guide
+[link=security-oidc.adoc]
```

Comment:
I think this should use xref instead of link (all of references), so it gets validated by build

Reply:
`link` is correct if used inside `CARD` component. Sample:

```
//DBClient
[CARD]
.DB Client
[icon=storage,link=dbclient.adoc]
--
Provides a unified, reactive API for working with databases in non-blocking way.
--
```